### DISCUSSION TITLE
BUG_can't submit transaction

### DISCUSSION THREAD

**Discussion by rewri3** [victorbalssa/abacus#123] [Bug] [answered]
**Describe the bug**
I can't submit a transaction because the submit button is already in a sending phase although it has not been pressed.
if I press the button nothing happens.

**To Reproduce**
1. Go to 'new transaction'
2. Scroll down to 'send button'
4. See that is in already in sending phase.

![Screenshot_20230827_104013_Abacus](https://github.com/victorbalssa/abacus/assets/121752414/5f22d645-bef6-484b-9206-3e3950b99791)

**Smartphone (please complete the following information):**
 - Device: S22 Ultra
 - OS: Android
 - Version 13

**Comment by victorbalssa** [comment]
Hi @rewri3,

This should be old local data in async storage after updating.

Could you go to `Settings` -> `Clear & Reset Application` and re-login, this will fix it.

Updating the app shouldn’t break anything, I'll take a look and deploy a hot fix if needed.

Thanks again for using Abacus!

**Reply by rewri3** [reply]
Hallo Victor, I had already reset the application but I did it a second time anyway. Unfortunately the problem persists.  When I create a new transaction the submit button is already in the submitting phase.

**Reply by rewri3** [reply, marked as answer]
Ok problem solved, I tried to delete all the application data but this time from the android app manager.  Now everything works!

**Reply by victorbalssa** [reply]
Thank you for the update.
